src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -435,6 +437,23 @@ public List<Map<String, String>> autoCompleteDatasets(String partial, Integer da
         ).collect(Collectors.toList());
     }
 
+    public Dataset approveDataset(Dataset dataset, User user, Boolean approvalBool) {
```

Comment:
minor naming nit, `approvalBool` is a boolean, so the `Bool` part of it seems redundant.